File: Java/Easy/3264.java
class Solution {
    public int[] getFinalState(int[] nums, int k, int multiplier) {
        int min;
        for (int i = 0; i < k; i++) {
            min = 0;
            for (int j = 1; j < nums.length; j++) {
                min = (nums[j] < nums[min]) ? j : min;
            }
            nums[min] *= multiplier;
        }

        return nums;
    }
}

File: Python/Easy/3264.py
class Solution:
    def getFinalState(self, nums: List[int], k: int, multiplier: int) -> List[int]:
        for i in range(k):
            min = 0
            for j in range(1, len(nums)):
                if nums[j] < nums[min]:
                    min = j
            nums[min] *= multiplier

        return nums
